indico/modules/events/abstracts/templates/management/_abstract_list.html

```diff
@@ -56,7 +56,10 @@
                             {% for item in dynamic_columns %}
                                 <th class="i-table">{{ item.title }}</th>
                             {% endfor %}
-                            <th class="i-table thin-column"></th>
+                            {% if reviewed_abstracts %}
```

Comment:
well, why do you need an empty column? If you haven't reviewed anything there will be nothing to show on that column. I'll post a screenshot of how it looks like.